Sources/App/Core/ExternalURL.swift

```diff
@@ -19,13 +19,15 @@ enum ExternalURL: URLRepresentable {
     case projectGitHub
     case projectSponsorship
     case raiseNewIssue
+    case raiseNewPackageListIssue
 
     var description: String {
         switch(self) {
             case .projectBlog: return "https://blog.swiftpackageindex.com"
             case .projectGitHub: return "https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server"
             case .projectSponsorship: return "https://github.com/sponsors/SwiftPackageIndex"
             case .raiseNewIssue: return "https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/new/choose"
+            case .raiseNewPackageListIssue: return "https://github.com/SwiftPackageIndex/PackageList/issues/new/choose"
```

Comment:
Should we perhaps link straight to "[add package](https://github.com/SwiftPackageIndex/PackageList/issues/new?assignees=&labels=Add+Package&template=add_package.yml&title=Add+%3CPackage%3E)" ?

```swift
case .addNewPackage: return "https://github.com/SwiftPackageIndex/PackageList/issues/new?assignees=&labels=Add+Package&template=add_package.yml&title=Add+%3CPackage%3E"
```

Reply:
We might even be able to pre-populate the url field via a query parameter. That'd be a _really_ convenient way to get new packages added.

Reply:
Done